### PR TITLE
fixed critical error while coping files from skeleton

### PR DESCRIPTION
**Pull request by S1wH**
fixed critical error while coping files from skeleton. There was problem while getting path to skeleton folder

### CODE REVIEW

File: cactus/bootstrap/package.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
-#coding:utf-8
+# coding:utf-8
+from cactus.cli import PATH
```
Comment (quillcraftsman) [inline cactus/bootstrap/package.py:2]: The constant "PATH" uses only in this file and not uses in cactus.cli. And cactus.cli it's top the top level module We have no reason to know about cactus.cli in bootstrap/package module.

I think we better way is to remove PATH in cactus.cli and put it into /bootstrap/package

File: cactus/skeleton/templates/base.html [hunk 2]
```diff
@@ -24,15 +24,9 @@
...
-       <li class="nav-item">
-                <a class="nav-link {% if_current_page 'index.html' 'active' '' %}" {% if_current_page 'aria-current="page"' '' %} href="index.html">Home</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link {% if_current_page 'about.html' 'active' '' %}" {% if_current_page 'aria-current="page"' '' %} href="about.html">About</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link {% if_current_page 'contact.html' 'active' '' %}" {% if_current_page 'aria-current="page"' '' %} href="contact.html">Contact</a>
-            </li>
+       <li class="{% if_current_page '/index.html' 'active' '' %}"><a href="{% url '/index.html' %}">Home</a></li>
```
Comment (quillcraftsman) [inline cactus/skeleton/templates/base.html:27]: class "nav-link" has been removed. It's bootstrap class to show menu properly.

File: cactus/bootstrap/package.py (hunk 1 above)
Comment (S1wH) [reply cactus/bootstrap/package.py:2]: i cant move PATH to /bootstrap/package due to importError, i can move it to utils/packaging

Comment (quillcraftsman) [reply cactus/bootstrap/package.py]: utils/packaging will be good. And delete in cli.py

**Pull request merged**